bower.json

```diff
@@ -23,7 +23,7 @@
     "paper-styles": "PolymerElements/paper-styles#^0.8.0"
   },
   "devDependencies": {
-    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.6",
```

Comment:
Yeah, but I made some backwards incompatible changes in `0.8.6` (`src` is not required in that version and beyond)